eng/pipelines/templates/steps/use-rust.yml

```diff
@@ -0,0 +1,19 @@
+parameters:
+  - name: Toolchain
+    type: string
+    default: stable
+
+steps:
+  - pwsh: |
+      $toolchain = '${{ parameters.Toolchain }}'
+      if ($toolchain -eq 'msrv') {
+          Write-Host "Reading MSRV from azure_core"
+          $toolchain = cargo metadata --manifest-path ./sdk/core/azure_core/Cargo.toml --format-version 1 | convertfrom-json | select -expand packages | where { $_.name -eq 'azure_core' } | select -expand rust_version
+      }
+
+      Write-Host "Setting RustToolchain variable to $toolchain"
```

Comment:
You changed the parameter name to `Toolchain` after our f2f conversation so this string is wrong, but I would change it to `Toolchain` everywhere. "Rust" is redundant since this is already a rust-specific file.

Reply:
The variable was still called RustToolchain.  I'll replace RustToolchain with Toolchain across the repo